src/styles/blocks/card.scss

```
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    margin: 0 3px 40px;
```

better not hardcode to center
```suggestion
    margin: 0 auto 40px;
```